### ISSUE TITLE
Couldn't send payment - "Pay" button non-responsive

### ISSUE THREAD

**Issue by fresheneesz** [ACINQ/phoenix#132]
I tried to send some money from a phoenix wallet to his phoenix wallet. We're both on android. He pressed received, I scanned the QR code, I changed the currency to 1 USD, and then pressed pay. Nothing happened. No error, just nothing. 

We then noticed his wallet said that since no money was in it, at least 10,000 satoshi must be sent. So ok, so I set it to 10,000 satoshi and press "pay". Again, nothing. I set it to 11,000. Nothing. Is there somewhere I can see developer error messages?

**Comment by dpad85** [comment]
You can see the application logs in Settings > Logs.

What's your device? and what android version ? Someone mentioned having issues when using a blue light filter app that somehow added an overlay over the UI and prevented interacting with it.

**Comment by fresheneesz** [comment]
>  Someone mentioned having issues when using a blue light filter app that somehow added an overlay over the UI and prevented interacting with it.

Oh, I do have one of those! Hmm, but why would it prevent just that button from being pressed? I haven't had any interference with any other app interaction that I've noticed. I'm using an app called "Blue Light Filter - Night Mode, Night Shift". 

My device is a Pixel 3 XL, android version 11. 

I did try without the filter and it did work. I haven't tried again with the filter on.
